spikeinterface/sorters/runsorter.py

```diff
@@ -187,15 +205,21 @@ def run_sorter_docker(sorter_name, recording, docker_image, output_folder=None,
             cmd = 'pip install --upgrade --force MEArec'
             commands.append(cmd)
 
-            cmd = 'pip install --upgrade --force https://github.com/SpikeInterface/spikeinterface/archive/master.zip'
+            cmd = 'git clone https://github.com/SpikeInterface/spikeinterface.git'
+            commands.append(cmd)
+            cmd = 'cd spikeinterface'
+            commands.append(cmd)
+
+            cmd = 'pip install .[full]'
+            commands.append(cmd)
+            cmd = 'cd ..'
```

Comment:
Tou should do
`pip install -e git+https://github.com/SpikeInterface/spikeinterface.git#egg=spikeinterface[full]`
it is more simpler

Reply:
I'll try that, thanks